lib/helpers/schema/getPath.js

```diff
@@ -26,10 +26,14 @@ module.exports = function getPath(schema, path) {
     schematype = schema.path(cur);
     if (schematype != null && schematype.schema) {
       schema = schematype.schema;
+      const currentPath = cur;
       cur = '';
```

Comment:
Instead of assigning a new value, how about just moving the `cur = ''` reassignment to the end of the block?

Reply:
Fixed in b91db2b, thanks for the suggestion :+1: